android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/FormUtils.java

```diff
@@ -1341,4 +1341,23 @@ protected void extractOptionOpenMRSAttributes(JSONArray valueOpenMRSAttributes,
         }
     }
 
+    public boolean checkValuesContent(JSONArray value) throws JSONException {
+        boolean showHiddenViews = true;
+        if (value.length() == 1) {
```

Comment:
its safer to use `opt...` instead of `get...` to obtain values from `JSONArray` or `JSONObject`

Reply:
Already checked the length to make sure index 0 is not null

Reply:
What if the JSONArray value is null

Reply:
That is checked where it's passed from.

Reply:
Refer to this https://github.com/OpenSRP/opensrp-client-native-form/blob/377dff2cac970cfa3686266ca3e2853c4d805f22/android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/ExpansionPanelFactory.java#L196